pyrfume/dbtables.py

```diff
@@ -0,0 +1,299 @@
+import datajoint as dj
+import quantities as pq
+from pyrfume import read_config
+import inspect
+
+dj.errors._switch_adapted_types(True)
+schema = None
+
+
+def update_schema():
+    init_schema()
+
+def init_schema():
+    schema_name = read_config("DATABASE", "schema_name")
+    context = globals()
+    global schema
+    schema = dj.schema(schema_name, context)
+    schema.context.update({'quantity_adapter': QuantityAdapter()})
+
+def drop_schema(force=False):
+    global schema
+    if not schema:
+        init_schema()
+    schema.drop(force=force)
+    schema = None
+
+def get_table(table_name):
+    global schema
+    if not schema:
+        init_schema()
+
+    result = None
+    if table_name:
+        try:
+            cls = globals[table_name]
+            if inspect.isclass(cls):
+                result = schema(cls)
+        except KeyError:
+            print("Class does not exist.")
+    
+    return result
+
+def get_tables():
+    global schema
+    if not schema:
+        init_schema()
+
+    result = {}
+    for key, value in globals().items():
+        if inspect.isclass(value) and value is not QuantityAdapter:
+            result[key] = (schema(value))
+    
+    return result
+
+
+class QuantityAdapter(dj.AttributeAdapter):
+    attribute_type = 'float'
+
+    def put(self, obj: pq.Quantity):
+        assert isinstance(obj, pq.Quantity)
+        obj = obj.rescale(pq.mL)
+        return obj.item()
+
+    def get(self, value: float):
+        return value * pq.mL
+
+
+class Molecule(dj.Manual):
+    definition = '''
```

Comment:
These classes do not need docstrings (only the functions above do) since the definition attribute can serve as a kind of documentation.